Header header
string name
float64 confidence
vision_msgs/BoundingBox2D boundingBox
